packages/app/src/cli/services/dev/processes/setup-dev-processes.test.ts

```diff
@@ -239,6 +243,101 @@ describe('setup-dev-processes', () => {
     })
   })
 
+  test('process list includes app polling when envVar is enabled', async () => {
```

Comment:
I've missed a bunch of conversation on this PR so this might've already been addressed, but did we decide to not include a test to show that the process list does not include polling when the var is disabled?

Reply:
@andrewhassan had a similar comment, see my response: https://github.com/Shopify/cli/pull/3918#discussion_r1619080831